.github/workflows/docs.yml

```diff
@@ -0,0 +1,26 @@
+name: Generate Documentation
+on:
+    push:
+        tags:
+            - "*"
+    workflow_dispatch:
+        inputs:
+            tag:
+                description: 'default tag for documentation'
+
+permissions:
+  contents: write
+
+jobs:
+    docs:
+        name: "Generate and Deploy Documentation"
+        uses: GoogleCloudPlatform/php-tools/.github/workflows/doctum.yml@doctum
```

Comment:
change to `@main` once https://github.com/GoogleCloudPlatform/php-tools/pull/150 is merged